recipe/meta.yaml

```
@@ -10,16 +10,16 @@ source:
   sha256: a14c1c5143081701d4fe62283e09cc5ad5336dff3a152de64eb77ee3180e9792
 
 build:
-  number: 0
+  number: 1
   noarch: python
 
 requirements:
   host:
     - python >=3.8
   run:
     - python >=3.8
-    - dask-core {{ version }}
-    - distributed {{ version }}
+    - dask-core ={{ version }},!={{ version }}a.*
+    - distributed ={{ version }},!={{ version }}a.*
```

Think there is no `.` in the current version scheme

```suggestion
    - dask-core ={{ version }},!={{ version }}a*
    - distributed ={{ version }},!={{ version }}a*
```

---

The `.` is here because we're using regex to match for pre-release versions (so essentially `{{ version }}a` followed by some variable amount of characters)

---

Yeah Conda's `*` is just a wildcard. Regex isn't implemented with version constraints AFAIK

https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications

---

Thanks for the context 🙂